ansible/configs/ocp4-workshop/software.yml

```diff
@@ -51,22 +51,25 @@
               aws_access_key_id = {{ hostvars.localhost.student_access_key_id }}
               aws_secret_access_key = {{ hostvars.localhost.student_secret_access_key }}
 
-        - name: Install Python Packages
+        - name: Ensure PIP and other packages are installed
           become: yes
           yum:
+            state: present
             name:
-            - https://gpte-public.s3.amazonaws.com/python2-botocore-1.6.0-1.el7.noarch.rpm
-            - https://gpte-public.s3.amazonaws.com/python2-s3transfer-0.1.10-1.el7.noarch.rpm
-            - https://gpte-public.s3.amazonaws.com/python2-boto-2.45.0-3.el7.noarch.rpm
-            - https://gpte-public.s3.amazonaws.com/python2-boto3-1.4.6-1.el7.noarch.rpm
-
-        - name: Install Packages
-          become: yes
-          package:
-            name:
+            - https://gpte-public.s3.amazonaws.com/python2-pip-8.1.2-9.el7.noarch.rpm
             - golang
             - unzip
 
+        - name: Install Python Packages
+          become: yes
+          pip:
```

Comment:
Wouldn't it be better to pin these to a specific versions?

Reply:
Well I couldn't use the ones that I had saved from EPEL - they were too old and AWS "stuff" was failing during the OpenShift install.

I could pin those again to the versions installed. But when AWS updates "things" and updates the packages we'd be left behind and would have breakage again.

Reply:
The alternative is a broken or "enhanced" library could be pushed to PyPI that doesn't work with our code. You are the primary owner of this config, so if you are comfortable essentially using 'latest' for these libraries, then ok. I guess there is challenges both ways, although I typically prefer to pin versions and deal with an update if the break backwards compatibility.

Let me know if you want to proceed as is.